content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -8,30 +8,108 @@ due to theft, damages, human errors, or other factors. As such, inventory adjust
 to reconcile the differences, and ensure that the recorded counts in the database match the actual
 counts in the warehouse.
 
-In Odoo, inventory adjustments can be accessed through :menuselection:`Inventory --> Operations -->
-Inventory Adjustments`.
+View inventory adjustments page
+===============================
 
-The :guilabel:`Inventory Adjustments` page shows all products that are currently in stock. Each
-line contains the following information:
+To view the :guilabel:`Inventory Adjustments` page, navigate to :menuselection:`Inventory app -->
+Operations --> Inventory Adjustments`.
 
-- :guilabel:`Product`
-- :guilabel:`Lot/Serial Number`
-- :guilabel:`On Hand Quantity`
+On the :guilabel:`Inventory Adjustments` page, all products that are currently in stock are listed,
+with each product line containing the following information:
+
+.. image:: count_products/count-products-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+- :guilabel:`Location`: is the specific location in the warheouse where a product is stored.
+- :guilabel:`Product`: is the product whose quantity is listed on the inventory adjustment line.
+- :guilabel:`Lot/Serial Number`: is the specific tracking identifier assigned to the specific
+  product listed. It can contain letters, numbers, or a combination of both.
```

Comment:
if there are many different SN's for one product, are they listed each in their own line? If so, maybe having a note to specify that or explain that might be helpful.

Reply:
Good note, I hadn't even thought of that! Turns out each SN will show up on its own product line. Added a note in to show that!